cloudbuild-site.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Why do we have a new config file with its contents as a subset of the original cloudbuild file? Could you elaborate a bit in the commit description?